javascript-conditionals/main.js

```diff
@@ -0,0 +1,75 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro,
+            recommendMovie
+ */
+function isUnderFive(number) {
+  return number < 5;
+}
+
+function isEven(number) {
+  return number % 2 === 0;
+}
+
+function startsWithJ(string) {
+  return string[0] === 'J';
+}
+
+function isOldEnoughToDrink(person) {
+  return person.age >= 21;
+}
+
+function isOldEnoughToDrive(person) {
+  return person.age >= 16;
+}
+
+function isOldEnoughToDrinkAndDrive(person) {
+  return false;
+}
+
+function categorizeAcidity(pH) {
+  if (pH >= 0 && pH < 7) {
+    return 'acid';
+  } else if (pH === 7) {
+    return 'neutral';
+  } else if (pH > 7 && pH <= 14) {
+    return 'basic';
+  } else {
+    return 'invalid pH level';
+  }
+}
+
+function introduceWarnerBro(name) {
+  switch (name) {
+    case 'yakko':
+      return "We're the warner brothers!";
+    case 'wakko':
+      return "We're the warner brothers!";
+    case 'dot':
+      return "I'm cute";
+    default:
+      return 'Goodnight everybody!';
+  }
+}
+
+function recommendMovie(genre) {
+  switch (genre) {
+    case 'action':
+      return 'Die Hard';
+    case 'comedy':
+      return '21 Jump Street';
+    case 'horror':
+      return 'Insidious';
+    case 'drama':
+      return 'The Notebook';
+    case 'musical':
+      return 'Pitch Perfect';
+    case 'sci-fi':
+      return 'Interstellar';
+  }
+}
```

Comment:
Missing a `default` clause here. Check out the example we gave you. What happens if someone passes in a genre that doesn't exist?